modules/admin-ui/src/main/webapp/login.html

```diff
@@ -78,6 +78,7 @@
     <script src="../../bower_components/angular-local-storage/dist/angular-local-storage.js"></script>
     <script src="../../bower_components/angular-wizard/dist/angular-wizard.js"></script>
     <script src="../../bower_components/angular-hotkeys/build/hotkeys.js"></script>
+    <script src="../../bower_components/angular-loading-bar/build/loading-bar.js"></script>
```

Comment:
Why would you want this in the login page? Mis-injected by grunt-wiredep?

Reply:
@staubesv it was injected by the auto-wiring. I'll remove it.